x-pack/plugins/transform/server/routes/api/transforms.ts

```diff
@@ -62,16 +97,24 @@ export function registerTransformsRoutes(routeDependencies: RouteDependencies) {
       }
     })
   );
-  router.get(
+
+  /**
+   * @apiGroup Transforms
+   *
+   * @api {get} /api/transform/transforms/:transformId Get transform
```

Comment:
This has probably been asked before, but why are these endpoints in this structure with `/api/transform/transforms`, whereas something like `/api/transform` would be in line with the back-end API? Don't suggest changing them in this PR, but just wondered what the background to this was.

Reply:
AFAIR it's a relict of the rename from `data_frame` to `transform`, the old endpoints were called something like `@api {get} /api/data_frame/transforms`. I agree we should revisit and clean up in a follow up.